docker_images/sklearn/tests/generators/iris.py

```diff
@@ -26,9 +28,15 @@ def push_repo(repo_name, local_repo):
     print(f"Pushing {repo_id}")
 
     client = HfApi()
-    client.delete_repo(repo_id, token=token)
+    try:
+        client.delete_repo(repo_id, token=token)
+    except RepositoryNotFoundError:
+        # repo does not exist yet
+        pass
     client.create_repo(repo_id=repo_id, token=token, repo_type="model")
 
+    # prevent AWS "503 Server Error: Slow Down for url" error
+    time.sleep(10)
```

Comment:
I've never had an issue here though. 10 seconds is a lot of time, maybe add like half a second if you really need to? This script never failed for me.

Reply:
I tried lower times and at 7 seconds it still failed, so I had to go to 10. I have no idea why this would cause the error, as we are just performing a few operations, but maybe they resolve to many S3 operations under the hood?

I agree that 10 sec is very long but since this script only needs to be called quite rarely, I thought we can live with that (exponential backoff would be more efficient).

Reply:
Sure, then we leave it as is for now.

cc @julien-c, I'm not sure if this is expected behavior.